src/components/search-form/search-form.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './search-form.css'
+
+const SearchForm = ({ initialValue, onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState(initialValue || '');
+  const placeholder = 'What do you want to watch?';
```

Comment:
If you want to hold some constant values, moving them out of the component is better. Otherwise, you just re-create them on each re-render, which is inefficient

Reply:
Placed SEARCH_PLACEHOLDER outside of the component